pkgs/top-level/aliases.nix

```
@@ -1165,6 +1165,7 @@ mapAliases {
   ibniz = throw "ibniz has been removed because it fails to compile and the source url is dead"; # Added 2025-04-07
   ib-tws = throw "ib-tws has been removed from nixpkgs as it was broken"; # Added 2024-07-15
   ib-controller = throw "ib-controller has been removed from nixpkgs as it was broken"; # Added 2024-07-15
+  ibus-kkc = throw "ibus-kkc has been removed as it depending on libkkc which was removed"; # added 2025-08-28
```

Shouldn't this be

```suggestion
  kkc = throw "ibus-kkc has been removed as it depending on libkkc which was removed"; # added 2025-08-28
```

since `all-packages.nix` had it exposed as `kkc`?